corehq/apps/registration/tests/test_views.py

```diff
@@ -1,12 +0,0 @@
-from mock import patch
-from django.core.urlresolvers import reverse
-from corehq.apps.domain.tests.test_views import BaseAutocompleteTest
-
-
-class TestNewWebUserRegistrationFormAutocomplete(BaseAutocompleteTest):
-
-    def test_autocomplete_enabled(self):
-        self.verify(True, reverse("register_user"), "full_name", "email")
-
-    def test_autocomplete_disabled(self):
-        self.verify(False, reverse("register_user"), "full_name", "email")
```

Comment:
re: https://github.com/dimagi/commcare-hq/pull/12796#issuecomment-240134603 by @biyeun 

Autocomplete of some fields on the signup page needs to be disabled for ICDS per requirements of the security audit. This is (was?) configured with `ENABLE_DRACONIAN_SECURITY_FEATURES` setting. The [test failure](https://github.com/dimagi/commcare-hq/pull/12796#issuecomment-240104659) seems like a real failure, and I think this test is not irrelevant.